.swi.yml

```diff
@@ -0,0 +1,14 @@
+version: 1
+builder:
+  configs:
+  - platform: ios
+    scheme: CustomDump
+  - platform: macos-xcodebuild
+    scheme: CustomDump
+  - platform: tvos
+    scheme: CustomDump
+  - platform: watchos
+    scheme: CustomDump
+  - documentation_targets: [CustomDump]
+    swift_version: 5.9
```

Comment:
It's better not to set a Swift version here unless you specifically want to stay on 5.9. We always use the latest released Version for docs if you leave this line off.

Reply:
Docs didn't generate, there must be something else up though - I'll investigate.

Reply:
https://github.com/SwiftPackageIndex/SwiftPackageIndex-Server/issues/2888